src/utils/dispute-utils.js

```diff
@@ -350,11 +360,11 @@ export function getAdjudicationPhase(dispute, round, date, courtConfig) {
  * @param {Object} courtConfig The court configuration
  * @param {Object} round The round to get the phases from
  * @param {Object} currentPhase The dispute's current phase
+ * @param {Number} currentTerm The court current term
  * @returns {Array} Array of all `round` phases.
  */
-function getRoundPhasesAndTime(courtConfig, round, currentPhase) {
+function getRoundPhasesAndTime(courtConfig, round, currentPhase, currentTerm) {
```

Comment:
Total bikeshedding, but if it's not too much trouble, maybe we can standardize on `courtConfig` always being the last parameter?